jujugui/static/gui/src/app/components/user-profile/kpi/kpi.js

```diff
@@ -0,0 +1,74 @@
+/*
+This file is part of the Juju GUI, which lets users view and manage Juju
+environments within a graphical interface (https://launchpad.net/juju-gui).
+Copyright (C) 2015 Canonical Ltd.
+
+This program is free software: you can redistribute it and/or modify it under
+the terms of the GNU Affero General Public License version 3, as published by
+the Free Software Foundation.
+
+This program is distributed in the hope that it will be useful, but WITHOUT
+ANY WARRANTY; without even the implied warranties of MERCHANTABILITY,
+SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Affero
+General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License along
+with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+'use strict';
+
+YUI.add('user-profile-entity-kpi', function() {
+
+  juju.components.UserProfileEntityKPI = React.createClass({
+
```

Comment:
`displayName` is a property in the object passed to `createClass` that defined the component name for debugging[0]. And @frankban is requesting a docblock for the component which outlines what it does and why.

[0] https://github.com/juju/juju-gui/blob/develop/jujugui/static/gui/src/app/components/spinner/spinner.js#L24